ci/run_playbooks.yml

```diff
@@ -81,6 +81,13 @@
             content: "{{ output.stdout }}"
             dest: "{{ logs_dir }}/ansible_run.log"
 
+
+    - name: Revert the version update
+      ansible.builtin.shell:
+        cmd: |
+          oc patch  openstackversions controlplane  --type json -p='[{"op": "replace", "path": "/spec/customContainerImages", "value": {} }]'
+      when: "{{ patch_openstackversions | bool }}"
```

Comment:
This looks reasonable.
My only concern is that if some tasks fail, that the patch won't be reverted.

It might be worth adding a handler or a try/except block to make sure this is always run.
A handler would be cleaner, but require some more testing, I think. 
You'd have to attach a handler to the patch task, and then it would be run at the end of the play, I think.
This would be a good follow-up learning task, and I'm not going to block merging this based on not having a handler.

Reply:
I will create a jira ticket for that